### ISSUE TITLE
--no-interaction appears to be implied

### ISSUE THREAD

**Issue by weierophinney** [RobLoach/docker-composer#102]
I've been experimenting with this layer to see if it would be usable in place of a local composer install, as well as as a container within a docker-compose specification. One thing I'm running into is that execution of it appears to imply the `--no-interaction` flag.

As an example, try the following:

```console
$ docker run --rm -u $UID -v `pwd`:/app composer/composer create-project zendframework/skeleton-application zf3-via-docker
```

This project includes a couple of composer plugins, zend-component-installer and zend-skeleton-installer, which each normally have prompts to allow the developer to provide information (in the former case, to potentially add application configuration; in the latter, to modify dependencies). In both cases, as well, running without interaction triggers default actions (usually: do nothing), or, detects that work has already been done and do nothing.

When run with this docker layer, however, I can see the plugins being triggered, but am never given the expected prompts.

I've run composer within layers previously (we include it with the default `Dockerfile` in the ZF skeleton application, for instance), and it runs normally, prompting as expected.

I've combed through this repo, and cannot see that the flag is being passed; is there perhaps some other configuration file I'm missing?

tl;dr: I'd like to run composer *with* interaction.

**Comment by alcohol** [comment]
You need to run the container with `-it` in that case.

Read https://docs.docker.com/engine/reference/run/#/foreground for more details.
